reana-serial.yaml

```diff
@@ -0,0 +1,31 @@
+version: 0.1.0
+metadata:
+  authors:
+   - Ana Trisovic <ana.trisovic@gmail.com>
+   - Lukas Heinrich <lukas.heinrich@gmail.com>
+   - Tibor Simko <tibor.simko@cern.ch>
+  title: ROOT6 and RooFit physics analysis example
+  date: 19 February 2018
+  repository: https://github.com/reanahub/reana-demo-root6-roofit/
+code:
+  files:
+   - code/gendata.C
+   - code/fitdata.C
+inputs:
+  parameters:
+    events: 20000
+    gendata: code/gendata.C
+    fitdata: code/fitdata.C
+outputs:
+  files:
+   - outputs/plot.png
+environments:
+  - type: docker
+    image: reanahub/reana-env-root6
+workflow:
+  type: serial
+  specification:
+    steps:
+      - environment: 'reanahub/reana-env-root6'
+        commands:
+        - 'echo "test"'
```

Comment:
See the README which `root` commands to put here.

We should make this default once the serial is committed and 0.3.0 released. (This will take some time...) We could then clean-up the top-level README to simplify workflow section and move Yadage and CWL parts into separate documentation.